ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -281,7 +309,12 @@ function getValues(pendingApproval, t, actions) {
             [t('blockExplorerUrl')]: pendingApproval.requestData
               .blockExplorerUrl,
           },
-          prefaceKeys: [t('networkName'), t('networkURL'), t('chainId')],
+          prefaceKeys: [
+            t('networkName'),
+            t('networkURL'),
+            t('chainId'),
+            t('currencySymbol'),
```

Comment:
Previously we didn't show the currency symbol in the `Allow this site to add a network` without the user clicking into `view all details`:
<img width="354" alt="Screen Shot 2022-07-11 at 1 20 26 PM (2)" src="https://user-images.githubusercontent.com/34557516/179263204-af82ac5d-54c3-485a-a4dc-a55a219a65a1.png">

now:
<img width="353" alt="Screen Shot 2022-07-15 at 11 05 37 AM" src="https://user-images.githubusercontent.com/34557516/179263250-def9eed0-d46b-47ec-8cec-76770580d8cc.png">